test_cephfs_sr.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from lib.common import wait_for, wait_for_not, vm_image
+import time
+
+# Requirements:
+# - one XCP-ng host >= 8.2
+# - remote cephfs mountpoint
+# - access to XCP-ng RPM repository from the host
+
+@pytest.fixture(scope='module')
+def host_with_cephfsprogs(host):
+    host.yum_install(['centos-release-ceph-jewel'])
+    host.yum_install(['ceph-common'])
+    yield host
+    # teardown
+    host.yum_remove(['ceph-common', 'centos-release-ceph-jewel'])
```

Comment:
This will not leave the test host in the same same as before the tests : many dependencies will be left on the host.

`yum-plugin-remove-with-leaves` may be a solution to clean-up

Reply:
I'll implement something